taxi/src/lib.rs

```diff
@@ -50,26 +50,34 @@ impl fmt::Display for Object {
 	}
 }
 
-type World = Vec<Vec<Object>>;
+/// Calculates distance between two points
+pub fn distance(p0: (u32, u32), p1: (u32, u32)) -> f64 {
```

Comment:
I'm actually not sure if this should be public, cause it also feels like unnecessary distraction

Reply:
There's strategies that involve using euclidean distance as reward function, so thought I'd have it to make it simple for people, but it's not necessary to solve the game so maybe you're right.